src/NexusMods.Telemetry/Events.cs

```diff
@@ -0,0 +1,51 @@
+using System.Diagnostics.CodeAnalysis;
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace NexusMods.Telemetry;
+
+[SuppressMessage("ReSharper", "UnusedMember.Global")]
+[SuppressMessage("ReSharper", "UnusedType.Global")]
+[SuppressMessage("ReSharper", "UnusedMember.Local")]
+public static class Events
+{
+    public static class Game
+    {
+        private const string Category = "Game";
+        public static readonly EventDefinition AddGame    = new(Category, Action: "Add Game");
+        public static readonly EventDefinition ViewGame   = new(Category, Action: "View Game");
+        public static readonly EventDefinition RemoveGame = new(Category, Action: "Remove Game");
+    }
+
+    public static class Loadout
+    {
+        private const string Category = "Loadout";
+        public static readonly EventDefinition CreateLoadout         = new(Category, Action: "Create Loadout");
+        public static readonly EventDefinition CreateLoadoutCopy     = new(Category, Action: "Create Loadout Copy");
+        public static readonly EventDefinition ApplyLoadoutChanges   = new(Category, Action: "Apply Loadout Changes");
+        public static readonly EventDefinition PreviewLoadoutChanges = new(Category, Action: "Preview Loadout Changes");
```

Comment:
There are unused events definitions here, is the plan to add the event submissions in future PRs or in this one?

Reply:
These events are mostly just WIP, @captainsandypants and I are still figuring out a list of events.

Reply:
Yeah, some of these seem a bit spammy, but I guess they are more an example for now